modules/effects/schematics-core/utility/change.ts

```diff
@@ -162,3 +166,13 @@ export function createChangeRecorder(
   }
   return recorder;
 }
+
+export function commitChanges(tree: Tree, path: string, changes: Change[]) {
+  if (changes.length === 0) {
```

Comment:
Only commit the file when there are changes, I forgot this in some of the v8 migrations making them slow (plus there was a information message on every file changes, even node_modules, which might confuse people).